emulate-browsers/chrome-80/index.ts

```diff
@@ -46,7 +46,7 @@ export default class Chrome80 {
 
   public readonly userAgent: IUserAgent;
   public readonly networkInterceptorDelegate: INetworkInterceptorDelegate;
-  public locale = 'en-US,en;0.9';
+  public locale = 'en-US,en';
```

Comment:
Does this not break the request headers?

Reply:
Not in DA.

Reply:
So if you set it this way, is Chrome still adding the q=0.9 that is on network requests? Or is the Double Agent test not checking Accept-Language?